package.json

```diff
@@ -133,6 +133,7 @@
         "setupFiles": [
             "<rootDir>/test/setupFile.js"
         ],
+        "maxWorkers": "60%",
```

Comment:
This was to help some flaky tests succeed. I'm not sure whether it helped, but I think it did.